server/hl7/hl7.controller.js

```diff
@@ -8,7 +8,19 @@ const storage = multer.diskStorage({
   filename: (req, file, cb) => (cb(null, `${file.originalname}-${Date.now()}`))
 });
 
-const upload = multer({ storage });
+const upload = multer({ storage,
+  fileFilter(req, file, cb) {
+    if (!file) {
+      cb(new Error('No file found'), false); //TODO: Change to return 400 or something if no file found
```

Comment:
instead of passing a normal `Error`, pass an `APIError` with the appropriate status code
```suggestion
      cb(new APIError('No file found', httpStatus.BAD_REQUEST), false);
```